src/workspace.coffee

```diff
@@ -47,6 +47,8 @@ class Workspace extends Model
       left: new PanelContainer({location: 'left'})
       right: new PanelContainer({location: 'right'})
       bottom: new PanelContainer({location: 'bottom'})
+      topBar: new PanelContainer({location: 'top-bar'})
+      bottomBar: new PanelContainer({location: 'bottom-bar'})
```

Comment:
[**@thomasjo**](https://github.com/thomasjo) commented on [24 okt. 2015 21:11 CEST](https://github.com/atom/atom/pull/9274#discussion_r42937216 "2015-10-24T19:11:10Z - Replied by Github Comment Enhancer"):
> They're not panels, they're (panel) containers.

I'm sorry, I meant when the content are panels.

I like the `top-most` and `bottom-most`, nut will leave that up to you guys. Just wasn't sure about `bar` when the containers aren't limited to bars only.